README.md

```diff
@@ -36,7 +56,9 @@ Your answer here.
 What is the purpose of the `<head></head>` tag in HTML?
 
 ```text
-Your answer here.
+The head element container for all head elements which include title of the page, scripts for
+the page, style information for the page, meta information, and other information that we haven't
+learned about yet.
 ```
```

Comment:
:+1:  used to contain any "rules", meta data, and other information that affects the webpage but does not directly show up on the webpage itself.